backend-models/service-tests/showDuggaservice-test.txt

```diff
@@ -0,0 +1,666 @@
+
+—------------------------------------------------------------------------------------------------------------------
+TEST #3 - 111-117
+update active users
+
+***
+Pre-req: $active != NULL
+***
+
+***
+Pre-req variables
+***
+
+
+$pre-values =
+{
+    “hjk4ert6”,
+    52
+};
+
+SQL-query: INSERT INTO groupdugga(hash,active_users) VALUES($pre-values[0], $pre-values[1])
+
+***
+* Get newly created active users, make sure it was added
+***
+	SQL-query: SELECT * FROM groupdugga WHERE hash=$pre-values[0]; //
+
+
+***
+* Update information on a active users
+* send parameters input to showDuggaservice.php
+***
+send
+{
+    $opt = “UPDATEAU”
+    $hash = “hjk4ert6”
+    $newToken = 9999
+}
+***
+Gather service output
+***
+Save the values of the echoed array as a JSON (at the end of the file. echo json_encode($array);)
+This is the expected output for the micro service.
+
+***
+* Remove inserted row from DB, using hash
+***
+
+	SQL-query: DELETE FROM groupdugga WHERE hash = $hash;
+
+—------------------------------------------------------------------------------------------------------------------
+—------------------------------------------------------------------------------------------------------------------
+***
+Test#4 - 261 - 309
+get data from userAnswer
+***
+
+***
+pre-req: !isSuperUser($userid) &&			isset($_SESSION["submission-$courseid-$coursevers-$duggaid-$moment"]) == TRUE &&
+isset($_SESSION["submission-password-$courseid-$coursevers-$duggaid-$moment"] == TRUE
+***
+Pre-querys:
+$preValuesCourse = {	
+                        9999
+                        1
+                    };
+
+INSERT INTO course(cid, creator) VALUES ($preValuesCourse[0], $preValuesCourse[1]);
+
+$preValuesListentries = {   9999, 
+                            "Inserttobedeleted",
+                            "UNK",
+                            4,
+                            12,
+                            2,
+                            1,
+                            1337,
+                            5019,
+                            1,
+                            1,
+                            0, 
+                            "UNK" }; 
+
+SQL-query:  INSERT INTO listentries(cid,entryname,link,kind,pos,creator,visible,vers,moment,gradesystem,highscoremode,feedbackenabled,feedbackquestion) 
+            VALUES ($preValuesListentries[0], $preValuesListentries[1], $preValuesListentries[2], $preValuesListentries[3], $preValuesListentries[4], $preValuesListentries[5], $preValuesListentries[6], $preValuesListentries[7], $preValuesListentries[8], $preValuesListentries[9], $preValuesListentries[10],$preValuesListentries[11],$preValuesListentries[12]);
+
+$pre-values = {
+                “ghj1jfg2”,
+                “dsa4cxz5”,
+                13,
+                9999,
+                5019
+            }
+
+INSERT INTO userAnswer (hash, password, variant, cid, moment) VALUES($pre-values[0], $pre-values[1], $pre-values[2], $pre-values[3], $pre-values[4])
```

Comment:
this insert does not work.
i believe you have done the easy mistake to presume that moment, in the table listentries actually is names moment (i did the same).
![image](https://user-images.githubusercontent.com/102600391/235078039-37019df5-12d1-48c5-b81c-114fa66f2fa7.png)
as you can see above, the values actuall name is lid
![image](https://user-images.githubusercontent.com/102600391/235078266-ba227522-4220-406d-8ee9-ba5f9fd412b9.png)